<!-- -->
<launch>

	<arg name="msf_localization_ros_node_name" 	default="msf_localization_ros_node" />

  <arg name="msf_localization_config_file" default="$(find msf_localization_ros)/configs/msf_localization_config_file.xml" />

	<arg name="robot_pose_topic_name" 	default="msf_localization/robot_pose" />
  <arg name="robot_pose_rate" 	default="50" />


  <node name="$(arg msf_localization_ros_node_name)" pkg="msf_localization_ros" type="msf_localization_ros" output="screen">

    <param name="msf_localization_config_file" value="$(arg msf_localization_config_file)" type="string"/>

    <param name="robot_pose_topic_name" value="$(arg robot_pose_topic_name)" type="string"/>
    <param name="robot_pose_rate" value="$(arg robot_pose_rate)" type="double"/>

  </node>


</launch>
